client/extensions/woocommerce/woocommerce-services/state/label-settings/actions.js

```diff
@@ -37,12 +37,7 @@ export const setFormMetaProperty = ( siteId, key, value ) => {
 	};
 };
 
-export const fetchSettings = siteId => ( dispatch, getState ) => {
-	const form = getLabelSettingsForm( getState(), siteId );
-
-	if ( form && ( form.data || form.meta.isFetching ) ) {
```

Comment:
Removed this condition, since a similar check is performed before the existing call to `fetchSettings`.